packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -216,13 +216,12 @@ function Products({ renderers, ...props }) {
 }
 
 Products.propTypes = {
-	renderers: React.PropTypes.shape({
+	Components: React.PropTypes.shape({
 		ActionDropdown: React.PropTypes.func,
 	}),
 };
 
-function HeaderBar(props) {
-	const renderers = getRenderers(props.renderers);
+function HeaderBar({ logo, brand, env, search, help, user, products, renderers }) {
```

Comment:
I don't see the benefit of spreading the argument here. It creates a ton of allocations. 
Please revert back to using the 'props'.